glue/plugins/scatter_group/client.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import print_function, division
+from abc import ABCMeta, abstractproperty, abstractmethod
+from functools import partial
+from pandas import DataFrame
+import numpy as np
+
+from ...external import six
+from ...utils import coerce_numeric, lookup_class
+from ...core.data import IncompatibleAttribute, ComponentID, Component
+from ...core.callback_property import (CallbackProperty, add_callback)
+
+from .util import get_colors
+from ...clients.scatter_client import ScatterClient
+from ...clients.layer_artist import (ChangedTrigger, ScatterLayerArtist)
+
+
+class ScatterGroupClient(ScatterClient):
+
+    """
+    A client class that uses matplotlib to visualize tables as scatter plots.
+    """
+    xmin = CallbackProperty(0)
+    xmax = CallbackProperty(1)
+    ymin = CallbackProperty(0)
+    ymax = CallbackProperty(1)
+    ylog = CallbackProperty(False)
+    xlog = CallbackProperty(False)
+    yflip = CallbackProperty(False)
+    xflip = CallbackProperty(False)
```

Comment:
You probably don't need to redefine most of these, since ScatterClient has them already